pumps/mongo_aggregate.go

```diff
@@ -48,8 +48,15 @@ type MongoAggregateConf struct {
 	// it will print an alert.
 	// Defaults to 1000.
 	ThresholdLenTagList int `json:"threshold_len_tag_list" mapstructure:"threshold_len_tag_list"`
-	// Determines if the aggregations should be made per minute instead of per hour.
+	// Determines if the aggregations should be made per minute (true) or per hour (false).
 	StoreAnalyticsPerMinute bool `json:"store_analytics_per_minute" mapstructure:"store_analytics_per_minute"`
+	// Determines the amount of time the aggregations should be made. Max (and default) = 60 - Min = 1.
```

Comment:
Let's rewrite this for something like: `Determines the amount of time the aggregations should be made (in minutes). It defaults to the max value is 60 and the minimum is 1.`